js/react/components/community/index.jsx

```diff
@@ -6,7 +6,7 @@ import Masonry, { ResponsiveMasonry } from 'react-responsive-masonry';
 import LoadingCard from '../shared/LoadingCard.jsx';
 
 function Community(props) {
-	const { user, _csrf } = props;
+	const { user, agency, _csrf } = props;
```

Comment:
what is the _csrf for?

Reply:
ITs a token to prevent cross site requests. Seems its set for a future use so i left it.